mapfishapp/jsbuild/build.sh

```diff
@@ -37,11 +37,11 @@ ${mkdir} -p ${releasepath} ${releasepath}/lang
    ${python} -m pip install virtualenv
  fi
  ${venv}/bin/jsbuild -h > /dev/null
- if  [ ! -d ${venv} ] || [ $? -eq 0 ]; then
+ if  [ ! -d ${venv} ] || [ $? -ne 0 ]; then
```

Comment:
why ?

Reply:
Because you want to install jsbuild only if it's not installed, and the result of calling `jsbuild -h` if it does not exist is error code 127.